db/migrate/20230401123456_add_new_fields_to_users.rb

```diff
@@ -0,0 +1,9 @@
+class AddNewFieldsToUsers < ActiveRecord::Migration[7.0]
+  def change
+    # Add new fields to the users table
+    add_reference :users, :blog, foreign_key: true
+    add_reference :users, :gift_card, foreign_key: true
+    add_reference :users, :user_chanel, foreign_key: true
```

Comment:
The 'add_reference' method is used for 'blog', 'gift_card', and 'user_chanel', but there is no 'user_chanel' model in the provided context. This could be a typo or an incorrect reference. Verify the model associations and update the migration accordingly.